ui-tests/src/test/resources/features/integrations/integration-import-export.feature

```diff
@@ -115,3 +117,62 @@ Feature: Integration - Import Export
 
     And "Camilla" navigates to the "Integrations" page
     Then she waits until integration "Integration_import_export_test" gets into "Published" state
+
+
+#
+#  2. integration-import from different syndesis instance
+#
+  @integration-import-from-different-instance
+  Scenario: Import from different syndesis instance
+
+    And "Camilla" navigates to the "Integrations" page
+    And Camilla clicks on the "Import" button
+
+    # import from resources TODO
+    Then Camilla imports integration from relative file path "src/test/resources/integrations/Imported-integration-another-instance-export.zip"
```

Comment:
java.lang.IllegalArgumentException: File not found

Reply:
Our gitignore has *.zip ignored and I forgot to force add the zip file. I fixed that but I am now blocked by mastepan's fill form refactor so do not merge yet, I added WIP label...